### PR TITLE
Removed email subject translations.

### PR DESCRIPTION
**Pull request by marksweb**
This removes translated email subjects to fix sentry error DJANGO-GIRLS-WEBSITE-E

Translation of emails would require a more complex process of storing a preferred language for a user and then making use of that in sending that user emails. So the translated stings here are pointless without that functionality.

### CODE REVIEW

**Comment by amakarudze** [comment]
Thanks @marksweb!

File: organize/emails.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 def send_application_confirmation(event_application):
```
Comment (amakarudze) [inline organize/emails.py:7]: So this file has this unused import which is causing flake 8 to fail `from django.utils.translation import gettext_lazy as _`. Could you kindly remove it? Thanks @marksweb

**Review by amakarudze — CHANGES_REQUESTED** [review]

File: organize/emails.py (hunk 1 above)
Comment (marksweb) [reply organize/emails.py:7]: Ah yes, of course. Too busy rushing things. 🤦‍♂️

**Comment by marksweb** [comment]
@amakarudze Fixed the unused import 👍

**Review by amakarudze — APPROVED** [review]

**Pull request merged**